tdms/tests/system/utils.py

```diff
@@ -132,7 +132,10 @@ def wrapped_function(*args, **kwargs):
     return func_decorator
 
 
-def run_tdms(input_filename: str, output_filename: str) -> None:
+def run_tdms(*args,
+             return_output: bool = False,
+             return_returncode: bool = False
```

Comment:
This is confusing if you specify both as `True`. I'd suggest just returning both as a tuple all the time.

Reply:
totally agree. shouldn't be so sloppy, even in tests!